src/DGtal/topology/doc/moduleDigitalTopology.dox

```diff
@@ -286,7 +286,7 @@ namespace DGtal {
   
  
  The resulting border can be visualized for instance by: 
- (see <a href="3dBorderExtraction_8cpp-example.html" >  3dBorderExtraction.cpp</a>)
+ (see <a href="3dBorderExtraction_8cpp_source.html" >  3dBorderExtraction.cpp</a>)
```

Comment:
I think we should better use `@ref` (or `\ref`) in order to get a more reliable link (and a different rendering too), like `(see @ref 3dBorderExtraction.cpp)`

Reply:
agree

Reply:
done